.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  vexchange-api-ci:
+    name: Vexchange API - CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
```

Comment:
```suggestion
      - name: Load .nvmrc
        run: echo ::set-output name=NVMRC::"$(cat .nvmrc)"
        id: nvm

      - name: Setup Node '${{ steps.nvm.outputs.NVMRC }}'
        uses: actions/setup-node@041bafb67276a76a9cc88cd8a4e99165e9eb287d
        with:
          node-version: '${{ steps.nvm.outputs.NVMRC }}'
```

Reply:
Phil has added a `.nvmrc` with `16.10.1` locked in

Reply:
@andersonaguiar do we have reasons to choose 14 over 16? If not let's just go with 16 since it's the new LTS